package exceptions_e_null_safety

fun testandoLancamentoExcecoes() {
    println("Só basta usar o throw e chamar o tipo da exceção, bem parecido com o Java")
    throw ExemploExcecao()
}